addons/road-generator/nodes/road_lane_agent.gd

```diff
@@ -37,6 +37,19 @@ var current_lane: RoadLane
 ## Cache just to check whether the prior lane was made visible by visualize_lane
 var _did_make_lane_visible := false
 
+enum MoveDir
+{
+	FORWARD = 1,
+	STOP = 0,
+	BACKWARD = -1
+}
+enum LaneDir
+{
+	RIGHT = 1,
+	CURRENT = 0,
+	LEFT = -1
+}
```

Comment:
Thanks for adding typing and docstrings 🙏 

My only request is to make sure this class enum doesn't clash with the one on RoadPoint which is named `LaneDir` already. Maybe this change be named `LaneChangeDir`? Though verbose, captures the intent fairly precisely.